metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarTest.java

```diff
@@ -285,6 +285,18 @@ public void testHappyPath() {
   }
 
   @Test
```

Comment:
I would add one more test that validates the behavior when you call LENGTH on an undefined variable; `LENGTH(foo)` when there is no variable `foo`.